client/web/src/App.tsx

```diff
@@ -65,9 +65,10 @@ export default function App() {
   const submitCheckpoint = async (
     previousVerificationId: string | null = null
   ) => {
+    setError(null);
     const sourceToken = await dodgeball.getSourceToken();
-
-    const endpointResponse = await axios.post(
+    try {
```

Comment:
Clear any previous errors on submit, and wrap submit in try/catch to capture any request errors (i.e. 404, 500 etc)